src/lib/config/defaultConfig.js

```diff
@@ -649,6 +649,10 @@ export const defaultConfig = {
     },
   },
   PATRONS: {
+    patronUniqueID: {
```

Comment:
IMHO, we should remove this from ILS, but you should create a PR in CDS Books to enable this make sure that it works correctly with CERN person ID